src/Shared/test/Shared.Tests/runtime/Http2/HPackDecoderTest.cs

```diff
@@ -151,25 +151,6 @@ public void DecodesIndexedHeaderField_DynamicTable()
             Assert.Equal(_headerValueString, _handler.DecodedHeaders[_headerNameString]);
         }
 
-        [Fact]
-        public void DecodesIndexedHeaderField_DynamicTable_ReferencedEntryRemovedOnInsertion()
```

Comment:
@amcasey just added this test in https://github.com/dotnet/aspnetcore/pull/57221. It should have been sync'd the other way - to dotnet/runtime.

@amcasey - would it be better if you first added tests like this to dotnet/runtime, and then let the code flow to aspnetcore?

cc @Tratcher - is this the preferred way to add tests to these shared files?